pkg/server/server.go

```diff
@@ -5,9 +5,11 @@ import (
 	"fmt"
 	"net/http"
 
+	httpSwagger "github.com/swaggo/http-swagger"
+
+	_ "github.com/CESARBR/knot-babeltower/docs"
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [a blank import should be only in a main or test package, or have a comment justifying it](https://app.codacy.com/app/joaoaneto/knot-babeltower/pullRequest?prid=4615084)